ADconnection.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bashfelfelfel
```

Comment:
The shebang line has been modified incorrectly to `#!/bin/bashfelfelfel`. Ensure it is corrected to `#!/bin/bash` to specify the correct path for the bash interpreter.